sourcegraph/sourcegraph
# Fix post-email verification return URL to /search

Now that the marketing site lives at https://sourcegraph.com, `/verify` that the link from the verification emails should point to should redirect the user to https://sourcegraph.com/search rather than the marketing site.

## Test plan

I did this:
- Send an email verification
- Click the link
- See if you get redirected to /search
It works! 🎉
